CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # App Center SDK for Unity Change Log
 
+## Version 2.6.1 (Under development)
+
+### App Center Push 
+
+#### iOS/UWP
+
+* **[Bug fix]** Fix missed push notification window if notification was received after the application closed.
```

Comment:
I don't understand what this window is about, our SDK does not provide any UI for push.

Reply:
window -> event

Reply:
```suggestion
* **[Bug fix]** Fix triggering notification event when restarting the application from a notification.
```
Or something along those lines.